Mathlib/Algebra/Group/Units/Equiv.lean

```diff
@@ -159,6 +159,24 @@ theorem divRight_eq_mulRight_inv (a : G) : Equiv.divRight a = Equiv.mulRight a
 
 end Group
 
+section CommGroup
+
+variable [CommGroup G]
+
+@[to_additive]
+lemma symm_divLeft (a : G) : (Equiv.divLeft a).symm = Equiv.divLeft a :=
+  ext fun _ ↦ inv_mul_eq_div _ _
+
+@[to_additive (attr := simp)]
+lemma divLeft_involutive (a : G) : Function.Involutive (Equiv.divLeft a) :=
+  fun _ ↦ div_div_cancel ..
+
+@[to_additive (attr := simp)]
+lemma const_div_involutive (a : G) : Function.Involutive (a / ·) :=
+  divLeft_involutive a
```

Comment:
I think this should go way earlier, given it can be proved with `fun _ ↦ div_div_cancel ..` and doesn't need aIf
```suggestion
@[to_additive (attr := simp)]
lemma const_div_involutive (a : G) : Function.Involutive (a / ·) :=
  fun _ ↦ div_div_cancel ..
```
and then `#find_home` to move this hopefully much earlier.